hw/ip_templates/rv_core_ibex/doc/boot-rom-patching.md

```diff
@@ -0,0 +1,459 @@
+# Boot, ROM execution and Patching
+
+## Glossary
+
+| Term  | Description                         |
+|-------|-------------------------------------|
+| SoC   | System On Chip                      |
+| DFx   | Design for Test and Debug           |
+| DFT   | Design for Test                     |
+
+## Scope
+
+This document covers the boot mechanism for the Integrated OpenTitan.
```

Comment:
It seems a bit strange to me that we're getting a copy of this in `top_earlgrey` and `top_englishbreakfast`! Is there anywhere sensible for the document to live that isn't in the template directory?

I realise that there are some technical problems with doing something like "Take a copy of this directory if XYZ is true" at the moment, but I'd suggest the better way to do things would be to only instantiate one copy (in a situation where it makes sense).